README.md

```diff
@@ -2,7 +2,10 @@
 
 This tool will take multiple IATI activity files and merge them into a single activity file.
 
-N.B. No validation is performed, nor is the `reporting-org` checked at present.
+Notes:
+- There is no explicit ordering to activities in the merged file
+- No validation is performed
+- The `reporting-org` checked for unique values.
```

Comment:
What should we say here? There is currently no checking against reporting-org and so it is possible to upload activity files from different organisations. If the merge utility checked for unique values that implies it would prevent the upload of files from the same reporting-org, which is the opposite of what we want.
I think this could read "The `reporting-org` is not checked for consistently and so currently it is possible to merge files from different publishers."